packages/webapp/src/components/Card/NoSearchResults/index.jsx

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Do we have a component similar to this one to display when there's only filters applied but no search term? I'm thinking I'll probably need one for my filters work, I can definitely add on top of this component though so no need to update it now

Reply:
As far I know, we haven't used a No Results component before this -- poking around it looks like the pattern has usually been just to return some text (e.g. in Crops: "No results found. Please change your filters.) Yes please feel free to add some more text options when you need them! 🙂